client/dashboard/sites/hosting-feature-activation-modal/error-content-info.tsx

```diff
@@ -0,0 +1,206 @@
+		),
+		variant: 'error',
+	},
+	[ EligibilityErrors.TRANSFER_ALREADY_EXISTS ]: {
+		message: __(
+			'Installation in progress. Just a minute! Please wait until the installation is finished, then try again.'
+		),
+		variant: 'info',
+	},
+	[ EligibilityErrors.NO_JETPACK_SITES ]: {
+		message: __( 'Try using a different site.' ),
+		variant: 'error',
+	},
+	[ EligibilityErrors.NO_VIP_SITES ]: {
+		message: __( 'Try using a different site.' ),
+		variant: 'error',
+	},
+	[ EligibilityErrors.SITE_GRAYLISTED ]: {
+		message: __(
+			'There’s an ongoing site dispute. Contact us to review your site’s standing and resolve the dispute.'
+		),
+		variant: 'error',
+		// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+		supportUrl: 'https://wordpress.com/support/suspended-blogs/',
+	},
+	[ EligibilityErrors.NO_SSL_CERTIFICATE ]: {
+		message: __(
+			'Certificate installation in progress. Hold tight! We are setting up a digital certificate to allow secure browsing on your site using "HTTPS".'
+		),
+		variant: 'info',
+	},
+};
+
+type HoldingMessage = {
+	title: string;
+	description: string;
+	supportUrl?: string;
+};
+
+const HoldingMessages: Partial< Record< EligibilityErrors, HoldingMessage > > = {
+	[ EligibilityErrors.NO_BUSINESS_PLAN ]: {
+		// TODO: Resolve plan name instead of hardcoding it.
+		title: __( 'Business plan required' ),
+		description: __( 'You’ll also get to install custom themes, plugins, and have more storage.' ),
+	},
+	[ EligibilityErrors.NON_ADMIN_USER ]: {
+		title: __( 'Site administrator only' ),
+		description: __( 'Only the site administrators can use this feature.' ),
+		// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+		supportUrl: 'https://wordpress.com/support/user-roles/',
+	},
+	[ EligibilityErrors.NOT_RESOLVING_TO_WPCOM ]: {
+		title: __( 'Domain pointing to a different site' ),
+		description: __(
+			'Your domain is not properly set up to point to your site. Reset your domain’s A records in the Domains section to fix this.'
+		),
+		// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+		supportUrl: 'https://wordpress.com/support/move-domain/setting-custom-a-records/',
+	},
+	[ EligibilityErrors.EMAIL_UNVERIFIED ]: {
+		title: __( 'Confirm your email address' ),
+		description: __(
+			'Check your email for a message we sent you when you signed up. Click the link inside to confirm your email address. You may have to check your email client’s spam folder.'
+		),
+	},
+	[ EligibilityErrors.EXCESSIVE_DISK_SPACE ]: {
+		// TODO: Implement upsells
+		title: __( 'Increase storage space' ),
+		description: __(
+			'Your site does not have enough available storage space. Please purchase a plan with additional storage or contact our support team for help.'
+		),
+	},
+	[ EligibilityErrors.IS_STAGING_SITE ]: {
+		title: __( 'Create a new staging site' ),
+		description: __(
+			'Hosting features cannot be activated for a staging site. Create a new staging site to continue.'
```

Comment:
This is interesting, the sentence does not make sense, if it can't be activated for a staging site (???) why is it suggesting to create a new staging site 😂

Reply:
I think I understand the message... it's meant to convey that the user cannot active hosting features for staging sites because in theory they should already be activated. Otherwise it's a broken staging site (i.e. Simple staging site). So the recommendation is to create a new staging site since this one is broken.